### ISSUE TITLE
Missing sendmail

### ISSUE THREAD

**Issue by aspiers** [mautic/docker-mautic#104]
Email delivery doesn't work out of the box because sendmail isn't included in the Dockerfile, so PHP mail can't find it and fails with a cryptic error:

    sh: 1: -t: not found                           

as explained at https://stackoverflow.com/a/44272240/179332

**Comment by aspiers** [comment]
Hmm, I tried installing sendmail and it still didn't work, so maybe I missed something else too.

**Comment by luizeof** [comment]
@aspiers I'd believe that sendmail via localhost is a good pratice... Mautic offers some best options like sendgrid, aws ses, mailjet.

**Comment by aspiers** [comment]
Yeah I have it working via SMTP to AWS SES now.  But switching to the SES API when https://github.com/mautic/mautic/pull/6977 lands will be even better.
